app/views/comments/_comment.html.erb

```diff
@@ -10,7 +10,7 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <% if current_user == comment.author || current_user == comment.photo.owner %>
+      <% if current_user == comment.author %>
```

Comment:
nice job hiding the edit feature for comments that don't belong to the current user